#! /usr/bin/env python
# Forearm navigate configuration

PACKAGE='ri_sensor_scanner_3d'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#       Name          		       Type         Reconfiguration level   Description   		Default     Min     Max
gen.add("goal_position",               int_t,       0,                      "goal position",    	0,          -30,    60)
gen.add("mode",                        bool_t,      0,                      "working mode",             False)
exit(gen.generate("scanner_3d", "scanner_3d", "scanner_3d"))
